test/e2e-image/e2e.sh

```diff
@@ -39,18 +39,34 @@ ginkgo "${ginkgo_args[@]}"               \
   -focus="${FOCUS}"                  \
   -skip="\[Serial\]|\[MemoryLeak\]"  \
   -nodes="${E2E_NODES}" \
+  --junit-report=report_e2e_test_suite.xml \
   /e2e.test
+echo "creating configMap report1 from the report file now.."
+kubectl create cm report1 --from-file report_e2e_test_suite.xml
+kubectl label cm report1 content=report
 
 echo -e "${BGREEN}Running e2e test suite with tests that require serial execution...${NC}"
 ginkgo "${ginkgo_args[@]}"   \
   -focus="\[Serial\]"    \
   -skip="\[MemoryLeak\]" \
+  --junit-report=report_e2e_test_suite_serial.xml \
   /e2e.test
+echo "creating configMap report2 from the report file now.."
+kubectl create cm report2 --from-file report_e2e_test_suite_serial.xml
+kubectl label cm report2 content=report
+
+
 
 if [[ ${E2E_CHECK_LEAKS} != "" ]]; then
   echo -e "${BGREEN}Running e2e test suite with tests that check for memory leaks...${NC}"
   ginkgo "${ginkgo_args[@]}"    \
     -focus="\[MemoryLeak\]" \
     -skip="\[Serial\]" \
+    --junit-report=report_e2e_test_suite_memleak.xml \
     /e2e.test
+  echo "creating configMap report3 from the report file now.."
+  kubectl create cm report3 --from-file report_e2e_test_suite_memleak.xml
```

Comment:
Maybe add on the cm name a more meaningful name

Reply:
Changed to hyphenated name, same as the file used to create the cm